src/Mod/Fem/femsolver/reportdialog.py

```diff
@@ -20,11 +20,12 @@
 # *                                                                         *
 # ***************************************************************************
 
-
-__title__ = "reportdialog"
+__title__ = "FreeCAD FEM reportdialog"
```

Comment:
`reportdialog` is ok? Or can it be `Report Dialog`?

Reply:
FreeCAD FEM solver report dialog